enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/document/DocumentProcessingService.java

```diff
@@ -202,6 +202,8 @@ public void checkDocumentSubmitWithProvider(OwnerId ownerId, DocumentResultEntit
         DocumentSubmitResult docSubmitResult;
         try {
             docsSubmitResults = documentVerificationProvider.checkDocumentUpload(ownerId, docVerification);
+            final IdentityVerificationEntity identityVerification = documentResultEntity.getDocumentVerification().getIdentityVerification();
+            auditService.auditDocumentVerificationProvider(identityVerification, "Check document upload for user: {}", ownerId.getUserId());
```

Comment:
This was tough architecture decision, but at the end, I believe that abstraction level is correct. Auditing should be done where the provider interface is called. Auditing must be same for whatever implementation, not only _ZenId_. Of course, we may improve logging in `ZenidDocumentVerificationProvider` or `ZenidRestApiService`, but if some details are important for, they should be propagated higher. Similar to #426

Reply:
Agree.